src/unix/solaris/mod.rs

```diff
@@ -966,10 +966,8 @@ extern {
                        serv: *mut ::c_char,
                        sevlen: ::socklen_t,
                        flags: ::c_int) -> ::c_int;
-    pub fn getpwuid_r(uid: ::uid_t,
-                      pwd: *mut passwd,
-                      buf: *mut ::c_char,
-                      buflen: ::size_t) -> *const passwd;
```

Comment:
It looks like the signature of getpwuid_r above is different than this one for solaris, was that intentional?

Unfortunately we don't have CI for the solaris bindings yet, so I'm not sure which one is right :(